src/Tracker.cpp

```diff
@@ -43,9 +45,8 @@ void Tracker::activateTracker(assets tracker) {
 void Tracker::end() {
   std::time_t endTimeStamp;
   std::time(&endTimeStamp);
```

Comment:
Those two lines are unused now that `SessionEndEvent` not longer takes the duration.